src/components/attendee/AttendeeLayout.module.scss

```diff
@@ -45,10 +45,11 @@ $-z-index--popover: layer.above($-z-index--ui);
 
   display: flex;
   flex-direction: column-reverse;
+  transition: filter 2s ease;
 
-  @media (max-width: breakpoint.$sm) {
+  @media (min-width: breakpoint.$md) {
     border-radius: 0;
-    top: 0;
+    top: 12px;
```

Comment:
another hardcoded value in pixels

Reply:
[ce03ad0](https://github.com/sparkletown/sparkle/pull/2973/commits/ce03ad0c063b92163c5039c84f70f34635278d0b)